### PR TITLE
[DOCFIX] Configure Alluxio with Secure HDFS

### PR DESCRIPTION
**Pull request by riversand963**
@aaudiber Could you take a look and comment? Thanks!

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 1]
```diff
@@ -152,6 +153,12 @@ Next, you can run a simple example program:
 $ bin/alluxio runTests
 ```
 
+For this test to succeed, you need to make sure that the login user of Alluxio cli, i.e. `bin/alluxio` has
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: I think you can remove the part which mentions `bin/alluxio`.

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 2]
```diff
@@ -152,6 +153,12 @@ Next, you can run a simple example program:
...
+read/write access to the HDFS directory mounted to Alluxio. If not specified in `./conf/alluxio-site.properties`,
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: I think the structure would more clear if you say something like:

By default, the login user is ... To change the login user, set ...

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 1 above)
Comment (riversand963) [reply docs/en/Configuring-Alluxio-with-secure-HDFS.md]: Done.

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 2 above)
Comment (riversand963) [reply docs/en/Configuring-Alluxio-with-secure-HDFS.md]: Done.

**Pull request merged**